tendrl/node_agent/monitoring/collectd/templates/gluster/tendrl_gluster.jinja

```diff
@@ -8,7 +8,7 @@
         graphite_host "{{ graphite_host }}"
         graphite_port "{{ graphite_port }}"
         peer_name "{{ fqdn }}"
-        provisioner {{ is_provisioner_node }}
+        node_id "{{ node_id }}"
```

Comment:
so now we will not set configuration to mark if the collectd node is provisioner or not. right?

Reply:
yes, instead of this we are directly reading from etcd.

Reply:
Ack